lib/iris/quickplot.py

```diff
@@ -139,6 +143,15 @@ def _label_1d_plot(*args, **kwargs):
     axes.set_ylabel(ylabel)
 
 
+def _footer(footer_text):
+    if footer_text:
+        text = plt.figtext(x=0.99, y=0.01, s=footer_text, fontsize=12, ha="right")
+        text.set_path_effects(
+            [patheffects.Stroke(linewidth=3, foreground="white"), patheffects.Normal()]
+        )
+        plt.subplots_adjust(bottom=0.15, left=0.1)
```

Comment:
Do you have an example of why this is needed?

Just curious ...

Reply:
...I can't replicate the original thing that made this necessary. I *do* think it looks better, do you foresee it causing problems?